playroom/frame-component.tsx

```diff
@@ -31,7 +31,7 @@ const App: React.FC = ({children}) => {
 
         ${isModalOpen ? 'body {overflow-y: hidden}' : ''}
 
-        *[class^='SplashScreen__'] {
+        *[class^='_1fu0koy1'] {
```

Comment:
@atabel I tried with this class that is more specific. I tried to create new branches and build with `npx vercel` and kept the same class name. We can try with this solution and see what happen when we include new components. If the classes changes, maybe we will have to roll back de playroom version